mkosi/__init__.py

```diff
@@ -7494,96 +7498,102 @@ def find_ovmf_vars() -> Path:
 
     die("Couldn't find OVMF UEFI variables file.")
 
+@contextlib.contextmanager
+def pre_run_qemu(args: MkosiArgs) -> [str]:
```

Comment:
Similar to above, let's name this context manager "run_qemu_cmdline()"

The type here is wrong (or maybe this is newer syntax? In that case, we need to support old python so we can't use newer typing syntax yet), It should be `Iterator[Sequence[str]]`